XCUITests/FxScreenGraph.swift

```diff
@@ -151,6 +153,10 @@ class Action {
     static let CloseTab = "CloseTab"
     static let CloseTabFromPageOptions = "CloseTabFromPageOptions"
     static let CloseTabFromTabTrayLongPressMenu = "CloseTabFromTabTrayLongPressMenu"
+
+    static let TypeEmail = "TypeEmail"
+    static let TypePassword = "TypePassword"
+    static let TapOnSignInButton = "TapOnSignInButton"
```

Comment:
For each of these actions, add `FxA` in the name somewhere.

Otherwise, it could type in an email in whatever text box is available. When it actually means: goto a specific place in the app and *then* type into the email box.